modules/transport-netty/src/main/java/org/elasticsearch/transport/NettyPlugin.java

```diff
@@ -85,6 +85,13 @@ public NettyPlugin(Settings settings) {
         );
     }
 
+    @Override
+    public Settings additionalSettings() {
+        return Settings.builder().put(super.additionalSettings())
```

Comment:
no need for the super call, it is just the abstract Plugin that returns an empty list